addons/account/static/src/xml/account_bank_statement_reconciliation.xml

```diff
@@ -172,6 +173,7 @@
                 <t t-if="line.debit !== 0" t-esc="line.debit_str"/><t t-if="line.credit !== 0" t-esc="line.credit_str"/>
                 <t t-if="line.amount_currency_str"> (<t t-esc="line.amount_currency_str"/>)</t>
             </td></tr>
+             <tr t-if="line.note"><td>Note</td><td><t t-esc="line.note"/></td></tr>
```

Comment:
Ok about the first note addition, but isn't this one a mistake ? I think this object has no note attribute : https://github.com/odoo/odoo/blob/8.0/addons/account/account_move_line.py#L760